docs/api/process.md

```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. Darwin for MacOS, linux for Linux and win32 for Windows.
```

Comment:
Please wrap the the values in backticks, e.g. `linux`. Also, `darwin` should be lowercase.

You might also consider using an unordered list to summarize the values for each platform.